s2clientprotocol/sc2api.proto

```diff
@@ -36,7 +36,7 @@ import "s2clientprotocol/ui.proto";
 //                  | ended (singleplayer only)                         | init_game             |
 // join_game*       | init_game (singleplayer or multiplayer host only) | in_game               |
 //                  | launched (multiplayer client only)                | in_game               |
-// restart_game     | ended (singleplayer only)                         | in_game               |
+// restart_game     | ended                                             | in_game               |
```

Comment:
Is the change to allow multiplayer restart in as well? I thought that was a separate change.